packages/styling/src/DeviceSettings/DeviceSettings-layout.scss

```
@@ -129,6 +129,8 @@
     background: var(--str-video__primary-color);
     border-radius: var(--str-video__border-radius-xs);
     transform-origin: left center;
+    // resting state; the live level is written inline by AudioVolumeIndicator
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)